package-generator/model/product_package.py

```diff
@@ -25,7 +25,7 @@
 import shutil
 import errno
 import os
-from util import utils
+from util import utils_file as utils
 from model.cookbook import Cookbook
 
 PACKAGES_FOLDER = "./../murano-apps"
```

Comment:
mm I don't like to define strings in a configuratino files. IMO configuration files are for values which can change, in this case, this value is not going to change. I agree in having a variable like: 
REPLACE_GE_NAME = {GE_name} or something like that. Take into account, these values should be in the Template files.